python/vowpalwabbit/sklearn_vw.py

```diff
@@ -672,6 +673,73 @@ class VWRegressor(VW, RegressorMixin):
 
     pass
 
+class ThresholdingMultiClassifierMixin(ClassifierMixin):
+
+
+    def predict(self, X):
+        """Predict class labels for samples in X.
+
+        Parameters
+        ----------
+        X : {array-like, sparse matrix}, shape = [n_samples, n_features]
+            Samples.
+
+        Returns
+        -------
+        C : array, shape = [n_samples]
+            Predicted class label per sample.
+        """
+        scores = self.decision_function(X)
+        indices = scores.argmax(axis=1)+1
+        return indices
+
+
+class VWMultiClassifier(SparseCoefMixin, ThresholdingMultiClassifierMixin, VW):
```

Comment:
this can then change to 
```
class VWMultiClassifier(ClassifierMixin, VW):
```

I don't think we're actually using SparseCoefMixin, and since that is moving location within sklearn we should just avoid it if possible..